File: problems1-100/p771.java
/*
Name: David Bruno
Start Date: 03/18/2019
File: p771.java

Solve Date: 03/18/2019
Runtime: 7 ms
Memory: 38 MB

*/

class Solution {
    public int numJewelsInStones(String J, String S) {
        int result = 0;
        
        for(char stone : S.toCharArray() ){
            if(J.contains(""+stone))
                result++;
        }

        return result;
    }
}

File: problems700-799/p771.py
# Name: David Bruno
# Start Date: 03/30/2019
# File: p771.java
#
# Solve Date: 03/30/2019
# Runtime: 36 ms
# Memory: 13.1 MB

class Solution:
    def numJewelsInStones(self, J: str, S: str) -> int:
        result = 0
        for stone in S:
            if stone in J:
                result+=1
        return result